### ISSUE TITLE
mouseButton not reporting correctly in 4.0b1 (Windows and Linux)

### ISSUE THREAD

**Issue by ppbrooks** [benfry/processing4#281]
## Description

mouseButton reporting 0 immediately or reverting to reporting 0 when either button is pressed

## Expected Behavior

mouseButton should report 37 when left button is down (and mousePressed == true)

## Current Behavior

When the left mouse button is pressed and held down, mouseButton often reports 0 and remains reporting 0.  Even if it starts reporting 37 (correctly), it reverts to 0 shortly thereafter.  Same behavior for right button.

No such problem in Processing-3.
## Steps to Reproduce

```processing
void setup() {
  size(200,200);
  frameRate(2);
}

void draw() {
  println(mousePressed, mouseButton);\
}
```

## Your Environment

 

* Processing version: 4 beta 1
* Operating System and OS version:  Windows 10 (64k)
* Other information:

## Possible Causes / Solutions

It looks like this problem has cropped up in previous versions, but it works correctly for me in Processing 3.

**Comment by benfry** [comment]
Confirmed working on macOS, broken on Windows and Linux… probably a regression in the changes for ctrl-click on macOS that introduced an oversight for the other platforms.

**Comment by benfry** [comment]
Fixed for 4.0 beta 2 with https://github.com/processing/processing4/commit/3257dae5d057d6b5738404d42b1accd70f69fa84
